Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java

```diff
@@ -411,19 +411,44 @@ private void updateUserPrivacySettings(final MaxAdapterParameters parameters)
         }
     }
 
-    private static BannerAdSize vungleAdSize(final MaxAdFormat adFormat)
+    private VungleAdSize vungleAdSize(
+        final MaxAdFormat adFormat,
+        final MaxAdapterParameters parameters)
     {
-        if ( adFormat == MaxAdFormat.BANNER )
+        final Map<String, Object> localExtraParameters = parameters.getLocalExtraParameters();
+        Object isAdaptiveObj = localExtraParameters.get( "adaptive_banner" );
+        boolean isAdaptiveBanner = false;
+        if ( isAdaptiveObj instanceof String ) {
+            try {
+                isAdaptiveBanner = Boolean.parseBoolean( (String) isAdaptiveObj );
+            } catch (Exception ignored) {
+            }
+        }
+        int customWidth = -1;
+        Object widthObj = localExtraParameters.get( "adaptive_banner_width" );
+        if (widthObj instanceof Integer) {
+            customWidth = (int) widthObj;
+        }
+        int customHeight = -1;
+        Object heightObj = localExtraParameters.get( "adaptive_banner_height" );
+        if (heightObj instanceof Integer) {
+            customHeight = (int) heightObj;
+        }
+        if ( !isAdaptiveBanner && customWidth != -1 && customHeight != -1 ) {
```

Comment:
Hi. I think I found a typo.
```suggestion
        if ( isAdaptiveBanner && customWidth != -1 && customHeight != -1 ) {
```

Reply:
Hi, I know the user guide has announced the `adaptive_banner` with true for adaptive banner and pubs can just only pass `adaptive_banner_width` for it. But for now we do not support adaptive banner, only custom banner size. That means we require both width and height for our inline placement.

Reply:
HI @Str4tos Updated the pr to be able to support adaptive and custome sizes.